modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -10,9 +10,9 @@
 
 [NOTE]
 ====
-The status of this Operator is General Availability for Amazon Web Services (AWS), {ibm-cloud-name}, global Microsoft Azure, Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
+The status of this Operator is General Availability for Amazon Web Services (AWS), Google Cloud Platform (GCP), {ibm-cloud-name}, global Microsoft Azure, Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
```

Comment:
Consider using the attribute for:

- Amazon Web Services (AWS): {aws-first}
- Microsoft Azure: {azure-full} -- though not sure if this one is needed. There isn't an attribute listed for Microsoft Azure Stack Hub, so not sure how beneficial an attribute for only part of the product name is. However, {ibm-cloud-name} Power VS suggests using {azure-full} Stack Hub would make things consistent.
- Google Cloud Platform (GCP) has an attribute: {gcp-first} 
- Alibaba Cloud also has an attribute: {alibaba}

Reply:
IBM Power VS actually has a sep attribute, so I have updated to use that. Azure Stack Hub should probably also have its own, so I will not do that one. Good looking out!